user_journey/views/cohort_analysis/clients_last_seen.view.lkml

```diff
@@ -0,0 +1,71 @@
+view: clients_last_seen {
+  derived_table:{
+    sql:
+      SELECT
+        *
+      FROM
+        `moz-fx-data-shared-prod`.search.search_clients_last_seen AS clients_last_seen
+      WHERE
+        {% condition cohort_analysis.date %} CAST(DATE_SUB(clients_last_seen.submission_date, INTERVAL 56 DAY) AS TIMESTAMP) {% endcondition %}
```

Comment:
This lets us pull in just one day of `clients_last_seen` for all data within the past 56 days, and otherwise, the day 56 days in the future.